### DISCUSSION TITLE
Welcome to kmtesters Discussions!

### DISCUSSION THREAD

**Discussion by Rochafella** [KMTesters/kmtesters#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Khurrm** [comment]
OK thanks for adding the discussion feature. Let us add more features in the coming weeks.

**Reply by Rochafella** [reply]
@Khurrm  Sure. thanks.

**Reply by Khurrm** [reply]
Next up is the setting of the main testing repos.
